### ISSUE TITLE
Ember 2.8.0 and getAttrFor

### ISSUE THREAD

**Issue by steveclarkin** [Panman8201/ember-bootstrap-switch#16]
Have just upgraded my project from Ember 2.7.3 to 2.8.0 and it looks like ember-bootstrap-switch broke.

Error:
Uncaught TypeError: this.getAttrFor is not a function

https://github.com/Panman8201/ember-bootstrap-switch/blob/master/addon/components/bootstrap-switch.js#L145

**Comment by Panman82** [comment]
_(sigh)_ They must have removed `getAttrFor` from ember, which _was_ technically private but useful. I use that in other addons so I'll have to look into it, but I think the `bootstrap-switch` plugin is dead. Unfortunately there [hasn't been any activity in over a year](https://github.com/nostalgiaz/bootstrap-switch/commits/master). Not sure when I'll get a chance to debug (I no longer use Bootstrap or this addon), but would welcome a PR.

**Comment by hoangdt84** [comment]
```
getAttrFor: function(attrs, key) {
    let val = attrs[key];
    return val && val.value !== undefined ? val.value : val;
  },
```

add this function into this component and it should work.

**Comment by dolgafar** [comment]
Cool.. works, i've created a PR for it.

**Comment by Panman82** [comment]
Fixed by #18
